### PR TITLE
Tweaks for 2017-03-28 data file

### PR DESCRIPTION
**Pull request by tadhg-ohiggins**
The sun in the sky, /
The ocean deeps, every wind: /
They all despair at Excel.

Address duplicate requirement IDs by appending zeroes to them; other tweaks to handle the latest data file.

### CODE REVIEW

File: reqs/management/commands/import_reqs.py [hunk 1]
```diff
@@ -238,12 +239,22 @@ def fix_excel_decimals(self, reqid):
         Since the rows are sequential (or should be!), we should be able to
         account for this.
         """
+        while reqid.endswith("."):  # Handle a hack around Excel problems
+            reqid = reqid[:-1]
```
Comment (cmc333333) [inline reqs/management/commands/import_reqs.py]: Super minor: Would
```python
reqid = reqid.rstrip('.')
```
be cleaner?

File: reqs/tests/import_reqs_tests.py [hunk 2]
```diff
@@ -81,7 +81,7 @@ def test_imports_correctly2(csv_file2):
     call_command('import_reqs', str(csv_file2))
 
     reqs = list(Requirement.objects.order_by('req_id'))
-    assert len(reqs) == 2
+    assert len(reqs) == 3
```
Comment (cmc333333) [inline reqs/tests/import_reqs_tests.py]: Minor -- should we be checking any information about the middle entry?

File: reqs/management/commands/import_reqs.py [hunk 3]
```diff
@@ -238,12 +239,22 @@ def fix_excel_decimals(self, reqid):
...
+        if "." not in reqid:
+            raise ValueError(
+                "Requirement ID without . separator: {0}".format(reqid))
         policy, requirement = reqid.split(".")
         last_policy, last_requirement = self.last_id.split(".")
         if policy == last_policy and last_requirement.endswith("9"):
             sequential = str(int(last_requirement) + 1)
             if sequential.rstrip("0") == requirement:
-                return "{0}.{1}".format(policy, sequential)
+                reqid = "{0}.{1}".format(policy, sequential)
```
Comment (cmc333333) [inline reqs/management/commands/import_reqs.py]: Do we need this logic (figuring out the sequent, checking if we're in sequence, saving the last req id, etc.) at all anymore?

Comment (tadhg-ohiggins) [inline reqs/management/commands/import_reqs.py]: I think we still need it. If we have the following IDs:
1.08, 1.09, 1.1, 1.11
then the sequential logic will correctly change the third one to `1.10`. If we remove it, it'll get imported as `1.1`, which won't be a duplicate, but will need correcting later.

File: reqs/tests/import_reqs_tests.py (hunk 2 above)
Comment (tadhg-ohiggins) [inline reqs/tests/import_reqs_tests.py]: I don't think we need to add that in this test; this was a hack around all three rows now getting imported since we're now not rejecting duplicates.
